win32ss/user/winsrv/consrv/frontends/gui/conwnd.c

```diff
@@ -2452,6 +2452,9 @@ ConWndProc(HWND hWnd, UINT msg, WPARAM wParam, LPARAM lParam)
 
             if (DefWindowProcW(hWnd /*GuiData->hWindow*/, WM_NCHITTEST, 0, lParam) == HTCLIENT)
             {
+                /* Don't display the context menu if QuickEdit is enabled */
+                if (Console->QuickEdit) break;
```

Comment:
In QuickEdit mode it should try pasting the text from clipboard when user clicks the right mouse button, probably this is not yet implemented.

Reply:
The paste code was already there.